src/index.js

```diff
@@ -0,0 +1,87 @@
+const tempDisplay = document.getElementById("temperatureDisplay");
+const skyDisplay = document.getElementById("topDisplay");
+const groundDisplay = document.getElementById("bottomDisplay");
+const cityInput = document.getElementById("mutableSubHeader");
+const bgDisplay = document.getElementById("gardenBackground");
+
+//document.getElementById("myDiv").style.backgroundColor = "lightblue";
+
+tempDisplay.textContent = 70;
+skyDisplay.textContent = "☀️ ☀️ ☀️ ☀️ ☀️ ☀️ ☀️ ☀️ ☀️";
+groundDisplay.textContent ="🌱 🌱 🌱 🌱 🌱 🌱 🌱 🌱";
+cityInput.textContent = "My Hometown";
+bgDisplay.style.backgroundColor = "orange";
```

Comment:
It's a best practice to avoid directly setting the style of an element in JS.  (This is because then we only have _one_ place we need to update styles when we change the theme of the site.)